controls/3.6/def.cf

```diff
@@ -278,11 +278,15 @@ bundle common def
       # This class is for PosgreSQL maintenance
       # pre-defined to every Sunday at 2 a.m.
       # This can be changed later on.
-
       "postgresql_full_maintenance" expression => "postgresql_maintenance_supported.Sunday.Hr02.Min00_05";
 
-      # Vacuum monitoring_mg table every day (except Sunday, when vacuum is run on entire db)
-      "postgresql_monitoring_maintenance" expression => "postgresql_maintenance_supported.!Sunday.Hr02.Min00_05";
+      # Vacuum full monitoring_mg table every day (except Sunday, when vacuum full is run on entire db)
+      # Required only by CFEngine Enterprise Hub 3.6.0 and 3.6.1
+      "postgresql_monitoring_maintenance" expression => "postgresql_maintenance_supported.!Sunday.Hr02.Min00_05.(nova_3_6_0|nova_3_6_1)";
```

Comment:
(enterprise_edition.(cfengine_3_6_0|cfengine_3_6_1)) instead of nova? If the semantics of nova/community change I think enterprise_edition will be an easier thing to replace.